examples/data-solutions/dp-foundation/06-sec-main.tf

```diff
@@ -0,0 +1,94 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# tfdoc:file:description Security project
+
+locals {
+  group_iam_sec = {
+    "${local.groups.data-engineers}" = [
+      "roles/dlp.reader",
+      "roles/dlp.user",
+      "roles/dlp.estimatesAdmin",
+    ],
+    "${local.groups.data-security}" = [
+      "roles/dlp.admin",
+    ],
+  }
+  iam_sec = {
+    "roles/dlp.user" = [
+      module.lod-sa-df-0.iam_email,
+      module.trf-sa-df-0.iam_email
+    ]
+  }
+  prefix_sec = "${var.prefix}-sec"
+}
+
+# Project
+
+module "sec-prj" {
```

Comment:
Is DLP always used? If not, we should probably avoid creating this extra project, and only create it if DLP or KMS are needed.

Reply:
DLP would be used at application layer: Dataflow pipeline or Bigquery (https://cloud.google.com/bigquery/docs/scan-with-dlp).

This is mentioned in the README, let me know if I should add additional context in the readme or in the tf code.